lighthouse-core/lib/sentry.js

```diff
@@ -108,20 +121,26 @@ function init(opts) {
       }
 
       return new Promise(resolve => {
-        Sentry.captureException(err, opts, () => resolve());
+        Sentry.withScope(scope => {
+          if (opts.level) {
+            // @ts-ignore - allow any string.
+            scope.setLevel(opts.level);
+          }
+          if (opts.tags) {
+            scope.setTags(opts.tags);
+          }
+          if (opts.extra) {
+            scope.setExtras(opts.extra);
+          }
+          Sentry.captureException(err);
+          resolve(); // TODO: idk ??
```

Comment:
todo: i believe this is all synchronous now

Reply:
I think you're right. We never await `Sentry.captureException` either (with the exception of tests)

Reply:
L85 suggests it's necessary.

```
// Special case captureException to return a Promise so we don't process.exit too early
```

Let's just keep it.

Reply:
Pretty sure it was just because the previous api was a callback one, and relied on some sort of event shenanigans which may not happen before the process closes (and Node will wait for a pending Promise to resolve before closing ... probably?)

changed to just have the function be `async` w/o using any promise stuff directly, just in case.